components/automate-chef-io/Makefile

```diff
@@ -3,7 +3,10 @@
 SHELL=bash
 SWAGGER_RESULT_FILE=static/api-docs/all-apis.swagger.json
 SWAGGER_DIR=data/docs/api_chef_automate
-SWAGGER_FILES = $(shell find $(SWAGGER_DIR) -name '*.swagger.json')
+#SWAGGER_FILES=$(shell find $(SWAGGER_DIR) -name '*.swagger.json')
+SWAGGER_FILES=data/docs/api_chef_automate/compliance/reporting/reporting.swagger.json data/docs/api_chef_automate/gateway/gateway.swagger.json
+STATIC_SWAGGER_FILES=$(shell find data/docs/api-static/ -type f | sort -n)
```

Comment:
Adding files to `data/docs/api-static/` is how I implemented adding things that can't come from the proto files (eg, reporting server's export endpoint) or would be awkward to put in a specific proto file (eg, the metadata that applies to the docs as a whole). They are sorted numerically so that things happen in a deterministic way.

Reply:
💡 This is a nice idea. I was wondering how to deal with those.